test/unit/email/emailVerifier.spec.js

```diff
@@ -14,12 +15,7 @@ chai.use(chaiHttp);
 chai.use(sinonChai);
 
 describe('Email Verifier', () => {
-  let stubList = [];
-
-  beforeEach(() => {
-    stubList.forEach(stub => stub.restore());
-    stubList = [];
-  });
+  afterEach(() => sinon.restore());
```

Comment:
Hello Luqman, kindly why you removed the stub array from the test.

Reply:
Thanks for reviewing @frostyblok. Every time we create `stubs` or `spies` using sinon they need to be cleaned up using the `.restore()` method.
It happens that I don't need to monitor all stubs being created, because sinon automatically adds all stubs to a default sandbox that can simply be cleaned up by calling `sinon.restore()`.

Please refer to this guide https://sinonjs.org/releases/v7.1.1/general-setup/

Reply:
It makes so much sense now, thanks for taking your time to explain. Good job, Luqman!